test/mocks/batch-event.js

```diff
@@ -0,0 +1,12 @@
+const event = require('./event')
+
+const FILENAME = 'batch-file.dat'
```

Comment:
should this be split out into a const?

Reply:
I'm not sure it needs to be, it's not used anywhere else but this file.

I could add a mock filename module, but feels like overkill.

Maybe if it was used across the tests that would be valuable.

Reply:
Change my mind to be consistent with other extension of mocks in tests